### PR TITLE
[TBOT-680]: Rewrite utils to typescript

### PR DESCRIPTION
**Pull request by aidera**
https://ghostdrive.atlassian.net/browse/TBOT-680

### CODE REVIEW

File: frontend/src/utils/anonymize.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export function anonymizeFullName(str) {
+export function anonymizeFullName(str: string) {
```
Comment (fifa-master) [inline frontend/src/utils/anonymize.ts:1]: return type?)

File: frontend/src/utils/vibration.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const vibrate = (
+  mode: 'light' | 'medium' | 'heavy' | 'rigid' | 'soft'
```
Comment (fifa-master) [inline frontend/src/utils/vibration.ts:2]: i think this type can be imported from telegram lib

File: frontend/src/config/video-file-extensions.ts [hunk 3]
```diff
@@ -16,7 +16,9 @@ export default [
   '.m4v'
 ];
 
-export const videoMediaTypes = [
+export default supportedFileTypes;
```
Comment (fifa-master) [inline frontend/src/config/video-file-extensions.ts:19]: use one type of export

File: frontend/src/config/docs-file-extensions.ts [hunk 4]
```diff
@@ -212,19 +211,21 @@ export default [
   }
 ];
 
-export const pdfMediaTypes = ['application/pdf'];
+export default supportedFileTypes;
```
Comment (fifa-master) [inline frontend/src/config/docs-file-extensions.ts:214]: use one type of export

File: frontend/src/config/audio-file-extensions.ts [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  '.weba', 
+  '.amr', 
+  '.ape', 
+  '.aiff', 
+  '.ac3', 
+  '.wma', 
+  '.m4a'
+];
+
+export default audioExtensions;
```
Comment (fifa-master) [inline frontend/src/config/audio-file-extensions.ts:20]: use one type of export

File: frontend/src/utils/editorjs/createNoteFile.ts [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  needEncryption?: boolean;
+  slug?: string | null;
+  uploadId?: string;
+}
+
+export const createNoteFile = async ({
+  name,
+  content = null,
+  noteSlug = null
+}: CreateNoteFilesProps) => {
```
Comment (fifa-master) [inline frontend/src/utils/editorjs/createNoteFile.ts:20]: return type

File: frontend/src/utils/vibration.ts (hunk 2 above)
Comment (aidera) [reply frontend/src/utils/vibration.ts:2]: image

File: frontend/src/utils/client.ts [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+export function isSafari(): boolean {
+  const userAgentString = navigator.userAgent.toLowerCase();
+  const chromeAgent = userAgentString.indexOf('Chrome') > -1;
+  let safariAgent = userAgentString.indexOf('Safari') > -1;
+
+  if (chromeAgent && safariAgent) safariAgent = false;
+  return safariAgent;
+}
+
+export function getBrowserName(): string {
```
Comment (fifa-master) [inline frontend/src/utils/client.ts:13]: create a type for return
string type bad

File: frontend/src/utils/string.ts [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+    return value;
+  }
+  return value?.match(/\d+/g)?.join('') || '';
+};
+
+export const capitalize = (value: string = ''): string => {
+  return value.charAt(0).toUpperCase() + value.slice(1);
+};
+
+export const fomatNumber = (num: number): string => {
```
Comment (fifa-master) [inline frontend/src/utils/string.ts:12]: correct spelling

**Review by fifa-master — APPROVED** [review]

**Pull request merged**